### ISSUE TITLE
Clarifying Graphic Information for "On Which Level Number is this place located?"

### ISSUE THREAD

**Issue by CurlingMan13** [streetcomplete/StreetComplete#6055]
**Use case**
When using StreetComplete to complete the quest "On Which level number is this place located?", it is often unclear to users, especially US-based users (with no OSM familiarity) which level to input.  For example, in the US we will say "The dentist is located on the 2nd floor", but in OSM, it is level=1.  level=0 starts at "Ground" floor on OSM.  In everyday conversation and reference, we call it the 1st floor.

I ran across many nodes at airports and other places where features are mistaken tagged with the wrong floor due to this confusion and the addition of a simple graphic or a clarification question mark would be useful.  They were being tagged one level higher than what they should be.

**Proposed Solution**
Add image to help clarify where level=0 begins to country's where the ground level is usually referred to as level=1 as opposed to OSM level=0.

![Proposed Quest Graphic](https://github.com/user-attachments/assets/b7bb2e90-22fd-40ba-8924-6a3e12a60650)

This is similar in nature to this issue, but instead designed for a different quest with fewer inputs and more confusion due to lack of a clarifying graphic.  I feel that because there is only one input for the quest (not 2) and being asked more for a business within a larger building, it may be helpful to clarify with some form of text.
https://github.com/streetcomplete/StreetComplete/issues/2661

StreetComplete is used by beginners who have never heard of OSM right alongside many who use this as merely a secondary method.  Many new users are not aware of the OSM nuance of level=* for floor tagging.  Many may be confused compared to building:levels=* or levels=* used in conjunction with building=*

**Comment by westnordost** [comment]
Heh, well, did you read the official wiki documentation?

https://wiki.openstreetmap.org/wiki/Key:level

**Comment by CurlingMan13** [comment]
It never occurred to me re-check the wiki.  Funny enough, it describes the **EXACT** point of confusion that I am wishing to try and address.

https://wiki.openstreetmap.org/wiki/Key:level#:~:text=Regardless%20of%20each%20country%27s%20local%20tagging%20convention%2C%20inexperienced%20mappers%20unaware%20of%20the%20convention%20may%20occasionally%20tag%20level%3D*%20based%20on%20the%20one%2Dbased%20system.

Perhaps another approach would be to have an info graphic button/link with (i) that can be selected with a more detailed graphic.  As it stands right now, there is no link to the wiki on this quest and no additional details explaining that level=0 is "ground floor".

I'm putting myself into the shoes of a new user who is not familiar with the wiki, may never look at the wiki and just contributes using StreetComplete.

**Comment by westnordost** [comment]
Well, my point is that due to in some regions (or for some buildings) `level=1` is ground floor, what value for the level should be chosen very much depends on how **other** POIs within the same building are tagged, and it should be consistent to **that**.

IIRC when I implemented the quest, the wiki didn't claim that actually `level=0` should always be ground level but stated that it is different in different countries or even buildings. But in the end, this doesn't matter much, because StreetComplete users cannot and should not "fix" the numbering in a whole building. Their answer should just be consistent with the numbering used in that building.

Could an explanation shown when expanding the (ℹ) help here? Do you have a suggestion for a wording?

**Comment by matkoniecz** [comment]
iff numbering should start from 0 also in say USA and it is accepted in general (and not just added by some wikifiddler) then it would be probably better to add 0-indexed numbers everywhere even if some POIs are tagged badly

or maybe SC should ask for and tag the signed numbers, and tag it in this separate tag?

**Comment by westnordost** [comment]
no, it is more important that it is consistent within one building than that the single POI one is editing is "correct". (In quotes, because whether that is more correct is debatable, looking at the de-facto current tagging situation.)

Separate tag (`level:ref`) would have the following problem: Currently, depending on the level selected, other nearby POIs on the same level are shown. This would work much less well with `level:ref`. Also, since `level:ref` is not necessarily a number, one would not be able to use the (+/-) control anymore.

No, really, we have this feature that allows clarifying how a question should be answered, let's use that. We just need a text that will help users that are confused by the question.

**Comment by westnordost** [comment]
No feedback, closing
